cvat/apps/engine/models.py

```diff
@@ -571,6 +571,13 @@ def list(cls):
     def __str__(self):
         return self.value
 
+class Manifest(models.Model):
+    filename = models.CharField(max_length=1024, default='manifest.jsonl')
+    cloud_storages = models.ManyToManyField('CloudStorage')
```

Comment:
Why ManyToMany?

Reply:
Because e.g `storage1` may have 2 manifests: `manifest.jsonl` and `sub1/manifest.jsonl`
and `storage2`may have 2 manifests: `manifest.jsonl` and `sub2/manifest.jsonl`

Reply:
They are not the same manifests, they are only the same name. The files themselves are different and stored in different locations

Reply:
So, why not OneToMany in CloudStorage class?